docs/methoddocs/alerts.md

```diff
@@ -5,3 +5,60 @@
     :members:
     :show-inheritance:
 ```
+
+```eval_rst
+.. autoclass:: py42.sdk.queries.alerts.alert_query.AlertQuery
+    :members:
+    :show-inheritance:
+```
+
+## Filter Classes
+
+The following classes construct filters for file event queries. Each filter class corresponds to a alert detail.
```

Comment:
"a" should be "an", 3rd to last word on this line

Reply:
yep